src/core/Action.js

```diff
@@ -5,7 +5,7 @@
 
 import { EventEmitter } from 'eventemitter3';
 import Ros from '../core/Ros.js';
-import { GoalStatus } from '../core/GoalStatus';
+import { GoalStatus } from '../core/GoalStatus.ts';
```

Comment:
This makes sense to me but could you double-check the resulting bundled code from Vite to make sure that this resolves okay for runtimes that _don't_ support TypeScript natively like Deno and ts-node do? I assume Vite will handle this, but I'm not 100% certain since I use extensionless paths on my projects.